lazy-pages/src/signal.rs

```diff
@@ -64,15 +63,16 @@ unsafe fn user_signal_handler_internal(
     info: ExceptionInfo,
 ) -> Result<(), Error> {
     let native_addr = info.fault_addr as usize;
-    let is_write = info.is_write.ok_or(Error::ReadOrWriteIsUnknown)?;
-    let wasm_mem_addr = ctx.wasm_mem_addr.ok_or(Error::WasmMemAddrIsNotSet)?;
+    let is_write = info.is_write.ok_or_else(|| Error::ReadOrWriteIsUnknown)?;
+    let wasm_mem_addr = ctx
+        .wasm_mem_addr
+        .ok_or_else(|| Error::WasmMemAddrIsNotSet)?;
 
     if native_addr < wasm_mem_addr {
         return Err(Error::OutOfWasmMemoryAccess);
     }
 
-    let offset =
-        u32::try_from(native_addr - wasm_mem_addr).map_err(|_| Error::OutOfWasmMemoryAccess)?;
+    let offset = (native_addr - wasm_mem_addr) as u32;
```

Comment:
Why did you do that? This moment is very important.

Reply:
Missed a moment with `usize`.